client/src/components/Header.jsx

```diff
@@ -0,0 +1,5 @@
+var Headers = () => {
+
```

Comment:
Your component name does not match the file name. You should make sure that they match up.